frontend/packages/dev-console/integration-tests/features/pipelines/pipelines-Triggers.feature

```diff
@@ -55,64 +58,63 @@ Examples:
+    Given Trigger is added to the pipeline "git-pipeline-triggerbinding" present in pipeline details page 
+    And user is at Event Listener Details page
+    When user clicks on Trigger Binding
+    Then user will be redirected to Cluster Trigger Binding Details page
     And user is able to see Details, YAML tabs
-    And Details tab is displayed with field names "Name", "Labels", "Annotations", "Created At", "Owner"
-    And Actions dropdown display on the top right corner of the page
+    And Details tab is displayed with field names Name, Labels, Annotations, Created At, Owner
+    And Actions dropdown display on Cluster Trigger Binding page
 
 
 @regression, @smoke
-Scenario: Remove Trigger popup details : P-09-TC10
-    Given user is at pipelines page
-    And pipeline is provided with trigger
-    When user selects "Remove Trigger" from the kebab menu
+Scenario Outline: Remove Trigger popup details : P-09-TC10
+    Given Trigger is added to the pipeline "<pipeline_name>" present in pipeline details page
+    And user is at pipelines page
+    When user selects "Remove Trigger" from the kebab menu for "<pipeline_name>"
     Then popup is displayed with header message "Remove Trigger"
-    And trigger template dropdown displayed with default selected option "Select Trigger Template"
-    And "Remove" button will be disabled
-
+    And trigger template dropdown displayed with help text Select Trigger Template
+    And Remove button will be disabled
 
-@regression, @smoke
-Scenario: Remove Trigger popup details : P-09-TC10
-    Given user is at pipelines page
-    And trigger is added to the pipeline
-    When user selects "Remove Trigger" from the kebab menu
-    Then popup is displayed with header message "Remove Trigger"
-    And trigger template dropdown displayed with default selected option "Select Trigger Template"
-    And "Remove" button will be disabled
+Examples:
+| pipeline_name              |
+| git-pipeline-removetrigger |
 
 
 @regression, @smoke
-Scenario: Remove the trigger from pipelines page : P-09-TC11
-    Given user is at pipelines page
-    And trigger is added to the pipeline
-    When user selects "Remove Trigger" from the kebab menu
-    And selet the first option from the Treigger Template drop down field
+Scenario Outline: Remove the trigger from pipelines page : P-09-TC11
+    Given Trigger is added to the pipeline "<pipeline_name>" present in pipeline details page
+    And user is at pipelines page
+    When user selects "Remove Trigger" from the kebab menu for "<pipeline_name>"
+    And selet the first option from the Trigger Template drop down field
     And user clicks on Remove button
-    Then popup get closed
-    And option "Remove Trigger" is not availale in kebab menu
+    Then option "Remove Trigger" is not availale in kebab menu for "<pipeline_name>"
+
+Examples:
+| pipeline_name                |
+| git-pipeline-removetrigger-1 |
 
 
 @regression, @manual
```

Comment:
Can you please remove the scenarios which are not required?

Reply:
We can use them in future, will not remove any of the scenarios as of now